docs/users/extend/custom-compose-files.md

```diff
@@ -26,6 +26,7 @@ version: '3.6'
 services:
   web:
     environment:
+      - PHP_IDE_CONFIG=serverName=mysite.ddev.site
```

Comment:
Instead of having to add it custom every time, couldn't we add it to the list of default env variables generated by ddev?

Reply:
BTW, I already add this customization to all my ddev containers, so +1 on its addition. The only question is about implementation.

Reply:
I like the idea, but I don’t think this can be automated because the value depends on what you‘ve set up for the server name in PhpStorm. It’s not the host name that ddev generates (though of course they could be the same).

Reply:
I guess what I'm suggesting is why not default it to the ddev generated name? Its what I've been using. Naming things is hard so I just use the name provided so I don't have to think of a new one.

Reply:
Note that PHP_IDE_CONFIG can easily be set in the bash session inside the container. That's my own preference. I just set it when I need it.  And yes $SITENAME (I think that's what's needed) would be better here than mysite.ddev.site.

Reply:
maybe PHP_IDE_CONFIG should be added as an env in the code, rather than having to do any of this.

Reply:
FWIW, I'm happy wherever it goes as long as it's documented.  I'd love it if this were deployed now since that's how it is right now, and then move the config to a better place, if needed, separately.